dojo/tools/anchore_engine/parser.py

```diff
@@ -1,59 +1,45 @@
-__author__ = 'jaguasch'
-
 import json
-from datetime import datetime
 
 from dojo.models import Finding
 
 
 class AnchoreEngineScanParser(object):
     def get_findings(self, filename, test):
-        tree = filename.read()
-        try:
-            data = json.loads(str(tree, 'utf-8'))
-        except:
-            data = json.loads(tree)
+        data = json.load(filename)
         dupes = dict()
-        find_date = datetime.now()
-
         for item in data['vulnerabilities']:
-            categories = ''
-            language = ''
-            mitigation = ''
-            impact = ''
-            references = ''
-            findingdetail = ''
-            title = ''
-            group = ''
-            status = ''
-            cve = None
+            cve = item.get('vuln')
```

Comment:
I think sometimes, there is `vuln` actually. Not sure anymore, but I would ensure this does not bomb.

Reply:
no bombing please

Reply:
No. can't happen. Vuln is the "id" of the data. Anchore use this field to link it to a CVE.
This specific line don't crash if vuln is not here compared to previous code with [] everywhere.